reference/ambassador-with-gke.md

```diff
@@ -81,7 +84,20 @@ spec:
     servicePort: 8080
 ```
 
-Now let's connect the other service from the tutorial to ambassador by specifying a mapping:
+Or 
+
+```yaml
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.